### ISSUE TITLE
[Bug] UTF-8 characters are not properly handled

### ISSUE THREAD

**Issue by Vayper** [mitre-attack/mitreattack-python#58]
## Expected Behavior

Handle properly import / export of comments including UTF-8 characters (french accents, for example)

## Actual Behavior

Non ASCII characters are scrambled.
For example, this comment : `Consiste à utiliser` is outputted `Consiste \u00c3\u00a0 utiliser `

## Steps to Reproduce the Problem

  1. Write by hand accents into a comment of the JSON layer file (for example: `Consiste à utiliser`)
  2. Load the layer using the API
  3. Export the layer using the API
  4. Open the outputted JSON file and see the comment

## Possible Solution

I solved this problem by making the following changes : 

1. https://github.com/mitre-attack/mitreattack-python/blob/c923d9e18fea263ea7cfaf56d1523d5dac837be7/mitreattack/navlayers/core/layer.py#L60
  changed to
  ```python
  with open(filename, 'r', encoding='utf-8') as fio:
  ```

3. https://github.com/mitre-attack/mitreattack-python/blob/c923d9e18fea263ea7cfaf56d1523d5dac837be7/mitreattack/navlayers/core/layer.py#L72
  changed to
  ```python
  with open(filename, 'w', encoding='utf-8') as fio:
  ```

4. https://github.com/mitre-attack/mitreattack-python/blob/c923d9e18fea263ea7cfaf56d1523d5dac837be7/mitreattack/navlayers/core/layer.py#L73
  changed to
  ```python
  json.dump(self.__layer.get_dict(), fp=fio, ensure_ascii=False)
  ```

**Comment by ArtificialErmine** [comment]
Good catch - I hadn't considered unicode characters and how they'd be handled by the library. I'll work on a patch on my end and let you know once it gets merged in.

**Comment by ArtificialErmine** [comment]
Resolved as of release v1.4.5.
